### PR TITLE
Add model person-vehicle-bike-detection-crossroad-yolov3-1020

### PR DESCRIPTION
**Pull request by LeonidBeynenson**

### CODE REVIEW

**Comment by LeonidBeynenson** [comment]
2 @eaidova :  
Ekaterina, could you take a look?

**Comment by eaidova** [comment]
Jenkins ok to test

**Comment by eaidova** [comment]
@LeonidBeynenson please add model to [index.md](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/models/intel/index.md)

**Comment by eaidova** [comment]
@eizamaliev  please take a look on model description

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+## Specification
+
+| Metric                          | Value       |
+|---------------------------------|-------------|
+| Mean Average Precision (mAP)    | 48.89%      |
+| AP people                       | 58.94%      |
+| AP vehicles                     | 62.05%      |
+| AP bikes/motorcycles            | 25.66%      |
+| Max objects to detect           | 200         |
```
Comment (eizamaliev) [inline models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md:29]: Where did this parameter come from? Original Yolo can produce much more detections.

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+| GFlops                          | 65.98       |
+| MParams                         | 61.92       |
+| Source framework                | Keras\*     |
+
+Average Precision (AP) is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+Validation dataset consists of 34,757 images from various scenes and includes:
```
Comment (eizamaliev) [inline models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md:38]: I think, you shouldn't use comma as digit separator. It makes values more SI-like.

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+|-----------------|--------------------------|
+| Vehicle         | 810,323                  |
+| Pedestrian      | 1,114,799                |
+| Bike/Motorcycle | 62,334                   |
+
+## Performance
+
+## Inputs
+
+1. name: `image_input` , shape: [1x3x416x416] - An input image in the format [BxCxHxW],
```
Comment (eizamaliev) [inline models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md:58]:
```suggestion
Name: `image_input` , shape: [1x3x416x416] - An input image in the format [BxCxHxW],
```

**Comment by eizamaliev** [comment]
@LeonidBeynenson please add model to `models.lst` of some demo.

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md (hunk 1 above)
Comment (LeonidBeynenson) [reply models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md:29]: Please, note that we evaluate on our crossroad test dataset (not COCO dataset)  
This is written below, where we describe the train/test datasets.

The original model has much worse result on our crossroad dataset.

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md (hunk 2 above)
Comment (LeonidBeynenson) [reply models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md:38]: OK

Comment (LeonidBeynenson) [reply models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md]: Done

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md (hunk 1 above)
Comment (eizamaliev) [reply models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md]: Nevertheless, I don't understand why you set max objects to 200. According to output description, it produces much more. Could you explain what it mean?

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-yolov3-1020.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Validation dataset consists of 34757 images from various scenes and includes:
+
+| Type of object  | Number of bounding boxes |
+|-----------------|--------------------------|
+| Vehicle         | 229503                   |
+| Pedestrian      | 240009                   |
+| Bike/Motorcycle | 62643                    |
+
+Similarly, training dataset has 219,181 images with:
```
Comment (eizamaliev) [inline models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-yolov3-1020.md:46]:
```suggestion
Similarly, training dataset has 219181 images with:
```

Comment (LeonidBeynenson) [reply models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-yolov3-1020.md]: Re-checked my datasets, updated numbers

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md (hunk 1 above)
Comment (LeonidBeynenson) [reply models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md]: I removed the line

Indeed, dataset contains not so many objects with height > 200, so this is less reliable, but nevertheless they may be detected

**Comment by LeonidBeynenson** [comment]
I remove person-vehicle-bike-detection-crossroad-yolov3-1020 from models.lst for object_detection_demo_yolov3_async to pass the tests

Please, add it back when the model is added to the package.

**Comment by LeonidBeynenson** [comment]
Jenkins please retry a build

**Comment by LeonidBeynenson** [comment]
Please, note that now the documentation build is failed, but I did not find errors concerning my changes

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-yolov3-1020.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
+# person-vehicle-bike-detection-crossroad-yolov3-1020
+
+## Use Case and High-Level Description
+
+This is a YOLO V3 network finetuned for Person/Vehicle/Bike detection for security
+surveillance applications. Works in a variety of scenes and weather/lighting
```
Comment (IRDonch) [inline models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-yolov3-1020.md:6]:
```suggestion
surveillance applications. It works in a variety of scenes and weather/lighting
```

**Comment by LeonidBeynenson** [comment]
2 @IRDonch , @vladimir-dudnik :     
I merged all the suggestions, but the GitHub UI shows that one requested change is not merged -- what should I do?  
Could you merge this PR?

File: models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-yolov3-1020.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function)
+    and multiply them by the corresponding anchors to get the absolute height and width values
+- `box_score` - confidence of detection box in [0,1] range
+- `class_no_1`,...,`class_no_80` - probability distribution over the classes in the [0,1] range,
+    multiply them by the confidence value `box_score` to get confidence of each class
+
+Since the model is finetuned on person/vehicle/bike detection dataset, it returns non-zero scores for the following classes:
+* person -- the first class score
+* non-vehicle (bike/motorcycle) -- the second class score
+* vehicle -- the third class score
```
Comment (eizamaliev) [inline models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-yolov3-1020.md:90]:
```suggestion
* person - the first class score
* non-vehicle (bike/motorcycle) - the second class score
* vehicle - the third class score
```

**Comment by LeonidBeynenson** [comment]
2 @vladimir-dudnik , @IRDonch :  Vladimir, Roman, could you merge this PR?

**Comment by LeonidBeynenson** [comment]
2 @IRDonch :     
NB: Eduard has just approved all the changes.

**Pull request merged**